### PR TITLE
moved cypress out of container

### PR DESCRIPTION
**Pull request by AndrewChubatiuk**
## What type of PR is this? 

- [x] Refactor

## Description

Moving cypress out of container decreases e2e test run up to 5 minutes, also for local setup no need to build container and install same deps. Also cypress builds from the start once there were changes in viz-lib, which it actually doesn't rely on.

- replaced seed-data command with `before` hooks in cypress
- moved cypress out of container
- replaced viz-lib linking with just a link

## How is this tested?

- [ ] Unit tests (pytest, jest)
- [ ] E2E Tests (Cypress)
- [ ] Manually
- [ ] N/A

### CODE REVIEW

**Comment by AndrewChubatiuk** [comment]
@justinclift @eradman could you please review?

**Comment by justinclift** [comment]
@AndrewChubatiuk I can't do it tonight (end of day for me already), and I probably can't do it tomorrow either (paperwork day).  Can look at it after that though, if no-one has gotten to it. :smile:

One question though, as it's not obvious from the PR description, but what's the benefit from moving Cypress out of a container?

I can understand upgrading old Cypress version to newer, that's a good thing. :smile:

I'm just unsure what the purpose/benefit/etc of removing the container bit is?  Note, that's not me arguing, I'm actually asking. :smile:

**Comment by justinclift** [comment]
> decreases e2e test run up to 5 minutes

Awesome, that sounds like a good improvement. :smile:

File: client/cypress/support/commands.js [hunk 1]
```diff
@@ -6,7 +6,106 @@ import "@testing-library/cypress/add-commands";
 
 const { each } = Cypress._;
 
-Cypress.Commands.add("login", (email = "admin@redash.io", password = "password") => {
+const LOGIN_NAME = process.env.CYPRESS_LOGIN_NAME || "Example Admin";
+const LOGIN_EMAIL = process.env.CYPRESS_LOGIN_EMAIL || "admin@redash.io";
+const LOGIN_PASSWORD = process.env.CYPRESS_LOGIN_PASSWORD || "password";
+const ORG_NAME = process.env.CYPRESS_ORG_NAME || "Redash";
+
+Cypress.Commands.add("setup", () => {
```
Comment (mirkan1) [inline client/cypress/support/commands.js:14]: How can I call this from terminal?
I got stuck so it never pass login screen on any test after I typed `npm run cypress all`

Comment (AndrewChubatiuk) [reply client/cypress/support/commands.js:14]: this cypress command can be called using `cy.setup`, it's added to client/cypress/support/commands.js
it's executed before e2e tests execution. you need to run `yarn cypress run-ci`

**Review by mirkan1 — CHANGES_REQUESTED** [review]

File: client/cypress/support/commands.js (hunk 1 above)
Comment (AndrewChubatiuk) [reply client/cypress/support/commands.js:14]: adding .env file is a responsibility of a developer. run "make .env"

**Pull request closed without merging**